### PR TITLE
ci(dependabot): remove reviewers config

### PR DESCRIPTION
**Pull request by aali309**
see what @andrewazores suggested: https://github.com/cryostatio/cryostat/issues/1660

### CODE REVIEW

**Comment by andrewazores** [comment]
This needs the `.github/CODEOWNERS` file created/updated to match.

**Comment by aali309** [comment]
> This needs the `.github/CODEOWNERS` file created/updated to match.

Should I also update the [release-drafter.yml](https://github.com/cryostatio/cryostat-reports/blob/main/.github/release-drafter.yml) and remove the `autolabeler` to match [https://github.com/cryostatio/cryostat/blob/main/.github/release-drafter.yml](https://github.com/cryostatio/cryostat/blob/main/.github/release-drafter.yml)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+pom.xml @cryostatio/maintainers
```
Comment (andrewazores) [inline .github/CODEOWNERS:1]: `src/main/docker/` should also be "owned" by maintainers team

**Comment by andrewazores** [comment]
> > This needs the `.github/CODEOWNERS` file created/updated to match.
> 
> Should I also update the [release-drafter.yml](https://github.com/cryostatio/cryostat-reports/blob/main/.github/release-drafter.yml) and remove the `autolabeler` to match https://github.com/cryostatio/cryostat/blob/main/.github/release-drafter.yml

Makes sense, go ahead and remove it

**Review by andrewazores — APPROVED** [review]

**Pull request merged**